pkg/webhook/webhook.go

```diff
@@ -86,6 +86,7 @@ type WebHook struct {
 	enableResourceQuotaEnforcement bool
 	resourceQuotaEnforcer          resourceusage.ResourceQuotaEnforcer
 	coreV1InformerFactory          informers.SharedInformerFactory
+	TimeoutSeconds                 *int32
```

Comment:
Nit: `timeoutSeconds` since it doesn't need to be exposed outside the package.

Reply:
Renamed the variable to be timeoutSeconds and also bumped the chart version to have the intermediate fix